### ISSUE TITLE
Implement SORT in the Repository

### ISSUE THREAD

**Issue by amunamun07** [anushkadhiman/ObjectTracking-DeepSORT-YOLOv3-TF2#4]
Hi @anushkadhiman  , If i wanna implement just the SORT and not DeepSORT from this repo. What are the changes that i should carry out?

**Comment by anushkadhiman** [comment]
Hi @amunamun07, both methods use the KF algorithm to predict the position of the objects in the current frame provided by the object detector.

The SORT method associates objects using bounding box detections to match measurements with predicted tracks, using the overlap of bounding boxes. On the other hand, to improve the bounding box association step, the Deep-SORT uses a CNN to extract appearance features from the object bounding box images.

Furthermore, instead of using IOU metric in SORT, the DeepSORT author uses Mahalanobis distance metric, bbox distance in measurement space as well as appearance distance. 

The implementation of both method are slightly different if you go through the theory.

**Comment by subbu04** [comment]
Hi.....
i want to use sort in tensorflow object detection  what are the changes i did

**Comment by subbu04** [comment]
Hi.....
actually, I used TensorFlow pre-train model  for object detection  in videos  now I want to track the frames using sort now what will be do
